src/main/cms/cms_menu_osd.c

```diff
@@ -132,7 +134,13 @@ OSD_Entry menuOsdActiveElemsEntries[] =
     {"ROLL PID",           OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_ROLL_PIDS], 0},
     {"PITCH PID",          OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_PITCH_PIDS], 0},
     {"YAW PID",            OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_YAW_PIDS], 0},
+    {"PROFILES",           OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_PIDRATE_PROFILE], 0},
     {"DEBUG",              OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_DEBUG], 0},
+    {"BATT WARN",          OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_MAIN_BATT_WARNING], 0},
+    {"DISARMED",           OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_DISARMED], 0},
+    {"PIT ANG",            OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_PITCH_ANGLE], 0},
+    {"ROL ANG",            OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_ROLL_ANGLE], 0},
+    {"ARMED TIME",         OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_ARMED_TIME], 0},
```

Comment:
Even without the suffix, being able to slash `osd_` off all the positions will be an improvement.

And yes, it will cause migration issues. But this stems to a big part from there being no standard way of naming things. If a standard format of `<pg name>_<parameter name>` was enforced, the end effect would be improved naming, e.g. `gyro_align`, `gyro_lpf`, and `gyro_moron_threshold`: No more guessing if its `align_gyro` or 'gyro_align', no more guessing what `moron_threshold` might belong to.

> To be honest I'm all for just removing this menu from the CMS, I'm not convinced there is enough of a use case for it in it's current state.

As a first step, we could make it conditional, so that it could be excluded for targets that are short on space. If this saves 1000 Bytes, that will come in handy in some cases.